src/vcpkg/postbuildlint.cpp

```diff
@@ -320,7 +320,31 @@ namespace vcpkg::PostBuildLint
 
         return LintStatus::SUCCESS;
     }
+    static LintStatus check_share_folder_names(const Filesystem& fs, const Path& package_dir)
+    {
+        const auto cmake_folder_name = package_dir / "share";
+        std::error_code ec;
+        const auto share_folders = fs.get_directories_non_recursive(cmake_folder_name, ec);
+        for (auto&& share_folder : share_folders)
+        {
+            auto folder_name = share_folder.filename().to_string();
+            const auto orig_folder_name = folder_name;
+            Strings::ascii_to_lowercase(folder_name.data(), folder_name.data() + folder_name.size());
 
+            if (orig_folder_name != folder_name)
+            {
+                vcpkg::printf(Color::warning,
+                              "Folders in /share must use lowercase ascii characters. Please rename it.\n"
+                              "    file(RENAME \"${CURRENT_PACKAGES_DIR}/share/%s\" "
+                              "\"${CURRENT_PACKAGES_DIR}/share/%s\")\n",
+                              orig_folder_name,
+                              folder_name);
```

Comment:
This message must be localized; please follow the instructions at https://github.com/microsoft/vcpkg-tool/blob/main/docs/localization.md.

Reply:
Ah I forgot this point, thanks for notice me!